data-azure-cosmos/src/test/groovy/io/micronaut/data/azure/AzureCosmosTestProperties.groovy

```diff
@@ -33,6 +33,7 @@ trait AzureCosmosTestProperties implements TestPropertyProvider {
         System.setProperty("javax.net.ssl.trustStoreType", "PKCS12")
 
         def defaultProps = [
```

Comment:
This is needed currently since emulator has not yet been updated with the new NonStreamingOrderBy query feature